guides/RELEASE-PROCESS.md

```diff
@@ -1,76 +1,53 @@
+      If there are expected visual changes, run `backstop approve` to save the new reference files. This should ideally be handled before the release process begins.
+   1. Run the release script to automatically begin the release process and publish to NPM.
       ```
-      git checkout -b v1.1.0
-      git add --all
-      git commit -m "Release v1.1.0"
-      git push --set-upstream origin v1.1.0
+      yarn release
       ```
+      This script will install dependencies, build design system assets, and run the linter and tests. If everything passes, `lerna` will prompt you for the new version number, which should follow the [SemVer release format](#versioning). After the versions are updated, the script will create a tagged release commit (prefaced with `core-`), a branch (prefaced with `release-`) containing the release commit, and a zip for the Github release notes.
+   1. The last step in the script will prompt you to continue with publishing to NPM. Proceed if everything is ready.
+   1. After the release script is completed, make sure to merge the release branch (i.e. `release-2.0.0`) into master.
 
-1. **Publish to NPM**
-
-   1. Log into NPM as `cmsgov` with `npm adduser`. Check your user account with `npm whoami`.
-
-      1. To use an access token, edit your `~/.npmrc` file so the contents are `//registry.npmjs.org/:_authToken={token}`
+1. **Create a release on GitHub**
 
-   1. Run the publish to NPM script.
-
-      ```
-      ./scripts/publish.sh
-      ```
+   1. [Draft a new release on GitHub](https://github.com/CMSgov/design-system/releases/new)
+   1. Tag the release with the newly created tag from the previous step (i.e. `core-2.0.0`).
+   1. Title the release using the release number (i.e. `2.0.0`)
+   1. Attach the release zip created from the previous step (i.e. `cmsgov-design-system-2.0.0.tgz`) to the release as an asset
+   1. Use the template below for the release notes
 
-      This will run `npm publish` for each public package in `packages/`.
-      **Note**: You should only publish the `master` branch to NPM. The publish script above will check out the `master` branch if it isn't currently the `HEAD`.
+   ```
+   ## 🚨 Breaking/Behavioral changes
+   ## 🚀 Added
+   ## 💅 Changed
+   ## 🛠 Fixed
+   ## 📦 Internal
+   ## 🚫 Deprecated
+   ```
 
-      **Note:** You will need to have your own NPM account and get access to the cmsgov NPM org in order to publish.
+   **Note**: View commits since the last release by going to the [releases page](https://github.com/CMSgov/design-system/releases) or by running: `` git log `git describe --tags --abbrev=0`..HEAD --oneline ``
 
-   1. Publish the release notes after the npm package is updated
+   1. Review the release notes and publish
 
 1. **Update the design.cms.gov documentation website**
```

Comment:
I would make this doc site step 2 and make the GitHub release notes step 3 so that the release notes are published after both the site is updated and the npm packages are published

Reply:
Hm, I could see it going either way. I originally ordered it this way because we need to go to Github after publishing to NPM anyway because we need to merge in the release commit, and because we need to use the tag created in step 1 in the release nots. Also, publishing the release notes with the attached file is required for the doc site download link to work.

Reply:
Ah gotcha, that makes sense. I happy to leave it in the order it is